### DISCUSSION TITLE
Rating: Adding a size Prop to PrimeVue Rating Component

### DISCUSSION THREAD

**Discussion by srhtak** [primefaces/community#2809] [PrimeVue]
I would like to propose adding a new size prop to the Rating component to improve user experience. This prop would allow users to easily adjust the size of the Rating component (e.g., small, medium, large). Currently, we have to manually adjust the size of the stars, which can lead to inconsistencies.

https://primevue.org/rating/

**Comment by j0rgedev** [comment]
I'd like to add that this size prop should be available in all the components 😁

**Reply by srhtak** [reply]
I think I will make my first open source contribution with prime. We are actively using this component in our project and we felt the lack of this component :D 

import * as React from 'react';
import { IconBase } from '../../iconbase/IconBase';

export const StarIcon = React.memo(
    React.forwardRef((inProps, ref) => {
        const pti = IconBase.getPTI(inProps);

        return (
            <svg ref={ref} width=“14” height=“14” viewBox=“0 0 14 14 14” fill=“none” xmlns=“http://www.w3.org/2000/svg” {...pti}>
                <path

Is it a correct method to send width height as props in svg?

**Comment by BenJackGill** [comment]
There are los of discussions around this but no traction on it yet. In an attempt to get more visibility to the problem here are some of the other discussions asking for a `size` prop on all form components:

- https://github.com/orgs/primefaces/discussions/2493
- https://github.com/orgs/primefaces/discussions/2684
- https://github.com/orgs/primefaces/discussions/2381
- https://github.com/orgs/primefaces/discussions/2809

I love PrimeVue! So hoping they can add this soon.
